tools/gromacs/energy.xml

```diff
@@ -0,0 +1,116 @@
+<tool id="gmx_setup" name="Extract energy components with GROMACS" version="@VERSION@">
+    <description>from an energy (EDR) file</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="requirements" />
+
+    <command detect_errors="exit_code"><![CDATA[
+
+        ln -s '$edr_input' ./edr_input.edr &&
+        echo $terms | tr , ' ' | gmx energy -f ./edr_input.edr -o ./energy.xvg
```

Comment:
@bgruening this works but I wonder if there is a nicer or more 'Galaxy-approved' way to do it.

The select parameter with `multiple="true"` gives me a comma-separated list and I want to turn it into a space-separated list and pipe it into `gmx energy`.

Reply:
Something like this: `' '.join($terms.split(','))`